.github/workflows/run-e2e-tests.yml

```diff
@@ -610,8 +643,9 @@ jobs:
     name: Get Chainlink image
     needs: [validate-inputs, load-test-configurations]
     if: ${{
```

Comment:
Can you perform the above logic here, and then skip the image build if its a public ECR, rather than throwing the logic into `load-test-configurations`. That or make a separate job completely.